docs/deploy.md

```diff
@@ -236,3 +237,281 @@ The target mainchain address of a backward transfer has to be a valid address: s
+topics: [
+'0x8c5be1e5ebec7d5bd14f71427d1e84f3dd0314c0f7b2291e5b200ac8c7c3b925',
+'0x00000000000000000000000010c6b6538860f3409bd6a16bbe7d6cb7bb371cbe',
+'0x000000000000000000000000f102765ba64849f77a2ac31a58e787b23480a6e0' ],
+data: '0x0000000000000000000000000000000000000000000000000000000000000001',
+logIndex: 0,
+blockHash: '0x9d0c7f96ce3bb91c489ed32116aa4832de188a8c32436d53aac54a326264e40f',
+args: [
+'0x10C6b6538860f3409Bd6a16BBe7d6CB7BB371cBe', '0xF102765ba64849f77A2AC31a58e787B23480a6E0', BigNumber { value: "1" },
+owner: '0x10C6b6538860f3409Bd6a16BBe7d6CB7BB371cBe', spender: '0xF102765ba64849f77A2AC31a58e787B23480a6E0', value: BigNumber { value: "1" }
+],
+decode: [Function (anonymous)],
+event: 'Approval',
+eventSignature: 'Approval(address,address,uint256)', removeListener: [Function (anonymous)],
+getBlock: [Function (anonymous)],
+getTransaction: [Function (anonymous)], getTransactionReceipt: [Function (anonymous)]
+} ]
+[ {
+'0x2d0a4bf9182bf77949b37e66492844558b8a0011bf2e6270cf781f2481b2b80e', address: '0x29E9954586d5248A162D1bC9e128175dcAe7402e', topics: [
+'0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef',
+'0x00000000000000000000000010c6b6538860f3409bd6a16bbe7d6cb7bb371cbe',
+'0x000000000000000000000000f102765ba64849f77a2ac31a58e787b23480a6e0' ],
+data: '0x0000000000000000000000000000000000000000000000000000000000000001',
+logIndex: 0,
+blockHash: '0x5d68062189c3dfd49ae3d0f99b6850f1a19d96dea82a2798e4b5706b7209c2c1',
+args: [
+'0x10C6b6538860f3409Bd6a16BBe7d6CB7BB371cBe', '0xF102765ba64849f77A2AC31a58e787B23480a6E0', BigNumber { value: "1" },
+from: '0x10C6b6538860f3409Bd6a16BBe7d6CB7BB371cBe', to: '0xF102765ba64849f77A2AC31a58e787B23480a6E0', value: BigNumber { value: "1" }
+],
+decode: [Function (anonymous)],
+event: 'Transfer',
+eventSignature: 'Transfer(address,address,uint256)', removeListener: [Function (anonymous)],
+getBlock: [Function (anonymous)],
+transactionIndex: 0, blockNumber: 48, transactionHa
+getTransaction: [Function (anonymous)],
+getTransactionReceipt: [Function (anonymous)] }
+]
+```
+
+### Interact with ERC20: EOA to EOA Transaction
+You can also interact with the ECR20 contract instance by making an EOA to EOA transaction. Copy the script (below) into your **tests/** folder:
+
+```
+const eoaToEoa = async () => {
+const receiver = "0xF102765ba64849f77A2AC31a58e787B23480a6E0"; console.log(
+`Receiver balance before: ${(
+await ethers.provider.getBalance(receiver)
+).toString()}` );
+const [sender] = await ethers.getSigners(); await sender.sendTransaction({
+to: receiver,
+value: ethers.utils.parseEther("1"), });
+console.log(
+`Receiver balance after: ${(
+await ethers.provider.getBalance(receiver) ).toString()}`
+); };
+eoaToEoa();
+```
+
+Run the command to interact with this contract instance:
+
+```
+npx hardhat test
+```
+
+You will see the following execution results:
+
+```
+Receiver balance before: 0
+Receiver balance after: 1000000000000000000
+```
+
+## Connect to a Dapp
```

Comment:
We spell `dApp` in a different page. Let's make sure it's consistent everywhere

Reply:
yes, made change from Dapp to dApp